src/System.IO/src/System.IO.csproj

```diff
@@ -81,9 +81,13 @@
     <TargetingPackReference Include="System.Private.CoreLib.WinRTInterop" />
     <TargetingPackReference Include="System.Private.Interop" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard13aot'">
     <TargetingPackReference Include="System.Private.Interop" />
+    <ValidateIgnoreReference Include="@(TargetingPackReference)" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'net462' and '$(TargetGroup)' != 'net463' and '$(TargetGroup)' != 'netcore50aot' and '$(TargetGroup)' != 'netstandard13aot'">
```

Comment:
I have simplified the condition - to match the pattern used in other places. I have also removed the early CoreRT netstandard13aot config - it was unused and bit-rotten.